metrics_utility/automation_controller_billing/report/report_ccsp_v2.py

```diff
@@ -143,7 +143,7 @@ def build_spreadsheet(self):
             self.wb.create_sheet(title='Indirectly Managed nodes')
             ws = self.wb.worksheets[sheet_index]
             indirects = job_host_summary_dataframe[job_host_summary_dataframe['managed_node_type'] == INDIRECT]
-            func(1, ws, indirects, managed_node_type='indirect')
+            self._build_data_section_usage_by_node(1, ws, indirects, managed_node_type='indirect')
```

Comment:
All the other if branches use `func`, based on the logic above, so this looks like a bug. Can you add a comment about it being intentional?

(I think I agree this does the right thing.)